### PR TITLE
Cve reputation bugfix

### PR DESCRIPTION
**Pull request by avidan-demisto**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14783

## Description
cveReputation script was returning inconsistent results because the cveSearch command executed in the script would sometimes return a 404 error. Added retry loop to the script to address the occasional 404 error.

## Required version of Demisto
4.1.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Code Review

## Additional changes
Also added a test playbook for this script.

### CODE REVIEW

File: Tests/id_set.json [hunk 1]
```diff
@@ -14270,6 +14270,24 @@
...
+            "cveReputation Test": {
+                "name": "cveReputation Test", 
+                "implementing_scripts": [
+                    "VerifyHumanReadableContains", 
+                    "cveReputation"
+                ]
+            }
+        }, 
+        {
+            "cveReputation Test": {
```
Comment (yaakovi) [inline Tests/id_set.json]: duplication: re-run the update_id_set.py script

File: Scripts/script-CveReputation.yml [hunk 2]
```diff
@@ -3,8 +3,19 @@ commonfields:
...
+  var resCmd = executeCommand('cve-search', {'cveId': args.input})[0];
+  var maxTries = 5;
+  var counter = 0;
+  while (isError(resCmd) && counter < maxTries) {
+      resCmd = executeCommand('cve-search', {'cveId': args.input})[0];
+      counter++;
+  }
+
+  if (isError(resCmd)) {
+      throw resCmd[0].Contents;
```
Comment (yaakovi) [inline Scripts/script-CveReputation.yml]: is this should be: 
```suggestion
       throw resCmd.Contents;
```

File: Tests/id_set.json (hunk 1 above)
Comment (yaakovi) [reply Tests/id_set.json]: this should resolve itself if you commit your code again

**Comment by yaakovi** [comment]
you have conflicts, please pull from master

**Comment by yaakovi** [comment]
you have missing tests.
you are missing releaseNotes for the script.
did you run the pre-commit hooks?

File: Tests/id_set.json (hunk 1 above)
Comment (avidan-demisto) [reply Tests/id_set.json]: I believe this is corrected now.

File: Scripts/script-CveReputation.yml (hunk 2 above)
Comment (avidan-demisto) [reply Scripts/script-CveReputation.yml]: Fixed as per your suggestion

File: Playbooks/playbook-Vulnerability_Handling_-_Nexpose.yml [hunk 3]
```diff
@@ -623,3 +623,6 @@ view: |-
   }
 inputs: []
 outputs: []
+releaseNotes: "Added 'No test' to playbook to allow for modified CveReputation script."
```
Comment (yaakovi) [inline Playbooks/playbook-Vulnerability_Handling_-_Nexpose.yml]: if the changes in the file are internal (tests, bug fixes) use the "-" mark as release notes

File: Playbooks/playbook-Vulnerability_Handling_-_Qualys.yml [hunk 4]
```diff
@@ -683,3 +683,6 @@ view: |-
   }
 inputs: []
 outputs: []
+releaseNotes: "Added 'No test' to playbook to allow for modified CveReputation script."
```
Comment (yaakovi) [inline Playbooks/playbook-Vulnerability_Handling_-_Qualys.yml]: same here

**Pull request merged**